src/styles/controls.plugin.scss

```diff
@@ -0,0 +1,51 @@
+.wikia-jw-small-player-controls {
+	.wikia-jw-small-player-controls-plugin {
+		align-items: center;
+		display: flex;
+		fill: white;
+		height: 100%;
+		justify-content: center;
+		left: 0;
+		opacity: 0;
+		position: absolute;
+		top: 0;
+		transition: all .1s linear .1s;
+		width: 100%;
+
+		div {
+			display: flex;
+			justify-content: space-evenly;
+			width: 100%;
+
+			svg {
+				cursor: pointer;
+				height: 35px;
+				width: 35px;
+			}
+		}
+	}
+
+	.jw-button-container {
+		display: none;
+	}
+
+	.jw-display-icon-container {
+		display: none;
+	}
+
+	.jw-controlbar {
+		margin-bottom: 2px;
+	}
+}
+
+.jwplayer:not(.jw-flag-user-inactive) {
```

Comment:
I'd keep this inside `.wikia-jw-small-player-controls` selector.  So it would like 
```
.wikia-jw-small-player-controls { 
  &:not(.jw-flag-user-inactive) {}
}
```